src/main/resources/config/application.yml

```
@@ -17,6 +17,14 @@ ingestion-flow-files:
       max-waiting-minutes: 5
       retry-delays-ms: 1000
 
+export-flow-files:
+  default:
+    filename-prefix: "\${EXPORT_FLOW_FILE_FILENAME_PREFIX:EXPORT}"
+  paid:
+    filename-prefix: "\${EXPORT_PAID_FLOW_FILE_FILENAME_PREFIX:EXPORT_PAID}"
+    page-size: "\${PAID_EXPORT_FLOW_FILE_SIZE_PAGE:20}"
```

let's use more greater page! or a huge export will take too much time and perform too many queries! (if you think, we allow until to 100k rows!)
```suggestion
    page-size: "\${PAID_EXPORT_FLOW_FILE_SIZE_PAGE:10000}"
```